packages/quantic/force-app/main/default/lwc/quanticSmartSnippetAnswer/__tests__/quanticSmartSnippetAnswer.test.js

```diff
@@ -13,7 +13,7 @@ const selectors = {
 };
 
 const exampleAnswer =
-  '<div><p>Example smart snippet answer</p><a href="#">Example inline link</a></div>';
+  '<div x-test_quanticsmartsnippetanswer=""><p x-test_quanticsmartsnippetanswer="">Example smart snippet answer</p><a href="#" x-test_quanticsmartsnippetanswer="">Example inline link</a></div>';
```

Comment:
New LWC-Jest update now adds these x-test_componentsName properties to elements in the HTML. Therefore the innerHTML was no longer matching

Reply:
Does it make more sense to change the test to check for example that there's a `div > p` with text matching `Example smart snippet answer` etc?